pages/builds/controlling_concurrency.md.erb

```diff
@@ -2,27 +2,61 @@
 
 Some tasks need to be run with very strict concurrency rules to ensure they don’t collide with each other. Common examples for needing concurrency control are deployments, app releases and infrastructure tasks.
 
-To help you control concurrency Buildkite provides two primitives: concurrency groups, and concurrency limits.
+To help you control concurrency Buildkite provides two primitives: concurrency limits, and concurrency groups. While these two primitives are interdependant, they are not explicitly linked.
```

Comment:
Defs yes to the extra one, but I'm a pretty intense fan of the oxford comma, so the one before the 'and' is sticking around.